examples/passport-login/README.md

```diff
@@ -7,27 +7,31 @@ A tutorial for implementing authentication in LoopBack 4 using
 - [Prerequisites](#prerequisites)
 - [Install](#install-the-example-locally)
 - [Tutorial - Facebook](#Try-it-out-with-FaceBook)
+- [Tutorial - Google](#Try-it-out-with-Google)
 
 ## Overview
 
 This example demonstrates how to use the LoopBack 4 features (like
 `@authenticate` decorator, strategy providers, etc) with
-[passport strategies](http://passportjs.org). It includes the local strategy as
-well as oauth2 strategies to interact with external OAuth providers like
-facebook, google,etc.
+[passport strategies](http://passportjs.org). It includes the OAuth2 strategies
+to interact with external OAuth providers like Facebook, Google,etc as well as
+local and basic strategies.
+
+You can use this example to see how to,
 
 - Log in or Sign up into a LoopBack App using passport strategy modules
 - Log in via external apps like Facebook or link those external profiles with a
   LoopBack user (for example, a LoopBack user can have associated
-  facebook/google accounts to retrieve pictures).
+  Facebook/Google accounts to retrieve pictures).
+- Use basic or local passport strategy modules
 
 ## Prerequisites
 
 Before starting this tutorial, make sure you have Client-ids/Secrets from third
 party apps
 
-- [facebook](https://developers.facebook.com/apps)
-- [google](https://console.developers.google.com/project)
+- [Facebook](https://developers.facebook.com/apps)
+- [Google](https://console.developers.google.com/project)
 - [twitter](https://apps.twitter.com/) **Not yet implemented**
```

Comment:
I'm wondering .... if we don't have this yet, maybe it's better to remove this line?

Reply:
facebook and google oauth is already implemented